### DISCUSSION TITLE
openers enhancement

### DISCUSSION THREAD

**Discussion by uberdude420** [Kav-K/GPTDiscord#77] [Ideas]
i officially request the implementation of a setting to allow for the openers.txt files to hold settings for p frequency presents temp and maybe token quantity. to better Taylor the way those prompts are received and interacted with by the api. i think this would go a long way to giving people ability to not just create pre prompts but Taylor it deeper with those settings  and as for noobs .. they don't know what those settings even do. so if a person sets up the bot properly in side of those txt files. it would make the experience a lot better and easier on those who don't know prompting but happen to be using the bot

**Comment by cherryroots** [comment]
Could make loading json files also a possibility with different fields for overrides and a field for the text itself

**Comment by Kav-K** [comment]
I think this was done!
